README.md

```diff
@@ -53,6 +54,17 @@ By supplying `--all` to c8, all files in directories specified with `--src` (def
 and `--exclude` flag checks, will be loaded into the report. If any of those files remain uncovered they will be factored
 into the report with a default of 0% coverage.
 
+## Source-Map support for pre-instrumented codebases
```

Comment:
Perhaps instead:

## Pre-instrumented vs., just-in-time instrumented codebases

### Source map files, vs., inline source maps

Just-in-time instrumented codebases will often insert source maps inline with the `.js` code they generate at runtime (_as an example, @babel-register/register can be configured to insert a source map foote).

Pre-instrumented codebases, e.g., running `tsc` to generate  `.js` in a build folder, may generate either inline source maps, or a separate `.map` file stored on disk.

`c8` can handle loading both types of source maps.

### Exclude after remap

... explain here why you when you would set the setting to `true`, vs., `false`.